openfisca_uk/parameters/benefit/pension_credit/minimum_guarantee.yaml

```diff
@@ -0,0 +1,23 @@
+description: Minimum income guarantee (basic amount) for Pension Credit
+SINGLE:
+  2015-04-06: 151.20
+  2016-04-06: 155.60
```

Comment:
isn't this 2016-04-11?

https://www.legislation.gov.uk/uksi/2002/1792/regulation/6/2016-04-11